src/py/aspen/covidhub_import/implementation.py

```diff
@@ -48,8 +48,10 @@
 logger = logging.getLogger(__name__)
 
 
-def covidhub_interface_from_secret(secret_id: str) -> CSqlAlchemyInterface:
-    interface = Cinit_db(Cutil.get_db_uri(secret_id))
+def covidhub_interface_from_secret(
+    covidhub_aws_profile: str, secret_id: str
+) -> CSqlAlchemyInterface:
+    interface = Cinit_db(Cutil.get_db_uri(secret_id, aws_profile=covidhub_aws_profile))
```

Comment:
does the leading C stand for covidhub? lol

Reply:
i feel like `CUtil` should be `c_util` and `Cinit_db` should be `c_init_db`, that seems more python-ish to me. That's a nitpick though

Reply:
I'll just be less lazy and call it covidhub_*.

Reply:
haha even better!